styles/_theme.scss

```diff
@@ -130,6 +130,14 @@ input[type=range][class~="clrS"] {
   @include textColor($emphasis2);
 }
 
+.clrTPriceAboveMarket {
+  @include textColor(#e4072b);
```

Comment:
Should this be hard coded, or use the error color? 

We may want it to be hard coded so it never changes if we later add theming. If so, the below market color should probably be hard coded below.

Reply:
I didn't use the error color because the error color is a slightly different color. We use `red` for that whereas the designs use `#e4072b` for the above market price. @morebrownies Should those colors be the same?

> We may want it to be hard coded so it never changes if we later add theming.

I was thinking the colors would have to be theme-able, otherwise if we hard code them and someone would theme up a background not conducive to the ones we hard code, it could completely wash them out.

Reply:
> @morebrownies Should those colors be the same?

Yeah sorry about that. Let's use the same red.